runtime/parachains/src/configuration.rs

```diff
@@ -77,6 +77,20 @@ pub struct HostConfiguration<BlockNumber> {
 	/// The delay, in blocks, before a validation upgrade is applied.
 	pub validation_upgrade_delay: BlockNumber,
 
+	// TODO: Which section do these belong to?
```

Comment:
I think we do not need to expose those to the parachains, at least not yet.

The closest is the `pvf_upgrade_delay`, but the idea is that parachains will look at the go-ahead signal anyway for synchronizing of the upgrades, so it is not needed as well.